.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Can we invoke with `yarn` instead of `npx`?

Reply:
I don't think so. `npx` runs the `bin`, but `yarn` runs the scripts in package.json